doc/whats-new.rst

```diff
@@ -54,6 +54,8 @@ Bug fixes
   By `Tom White <https://github.com/tomwhite>`_ and `Kai Mühlbauer <https://github.com/kmuehlbauer>`_.
 - Ensure dtype of reindex result matches dtype of the original DataArray (:issue:`7299`, :pull:`7917`)
   By `Anderson Banihirwe <https://github.com/andersy005>`_.
+- Fix bug where empty zarr ``chunk_store``s were were ignored as if they were ``None`` (:pull:`7923`)
```

Comment:
```suggestion
- Fix bug where an empty zarr ``chunk_store`` were ignored as if they were ``None`` (:pull:`7923`)
```
Looks like some formatting issue here.

Reply:
Thank you for catching the double `were`!